crawler/rabbit/async_consumer.py

```diff
@@ -90,7 +119,7 @@ def on_connection_open_error(self, _unused_connection, err):
         :param pika.SelectConnection _unused_connection: The connection
         :param Exception err: The error
         """
-        LOGGER.error("Connection open failed: %s", err)
+        LOGGER.error("Connection open failed: %s", AsyncConsumer._reap_last_connection_workflow_error(err))
```

Comment:
I had to do this otherwise the connection failure becomes a blank string.  Not sure why Pika chose this approach.

Reply:
Maybe add a comment about this?

Reply:
Fair enough.  I'll do that straight onto `develop` since I've merged this.